pages/juang961225/style.css

```diff
@@ -0,0 +1,115 @@
+body {
+    display: flex;
+    flex-wrap: nowrap;
+    margin: 0px;
+    padding: 0px;
+}
+.main {
+    align-items: flex-start;
+    background-image: url(../juang961225/pictures/FOR\ THE\ HORDA.jpg);
+    background-position: center;
+    background-size: contain;
+    display: flex;
```

Comment:
Hey buddy you have duplicated this line.

Reply:
solved